pkg/tempo/yaml/tempoStack.yaml

```diff
@@ -0,0 +1,19 @@
+kind: TempoStack
+apiVersion: tempo.grafana.com/v1alpha1
+metadata:
+  name: sample
+spec:
+  resources:
+    total:
+      limits:
+        cpu: 2000m
+        memory: 4Gi
+  storage:
+    secret:
+      name: my-storage-secret
+      type: s3
```

Comment:
If your OCP cluster is on AWS , then the storage is using s3. However, if your cluster is on Azure or Openstack, this type s3 will not work.

Reply:
It uses minio, not s3 on aws directly.
I have tried to run that test on OCP on Azure ( and on ARO  and it works on both clusters.